README.md

```
@@ -2,9 +2,9 @@
 
 # F1 Telemetry Manager
 
-### *FastAPI backend and Streamlit post-race UI for F1 StratLab.*
+### *FastAPI backend and React web app (post-race UI) for F1 StratLab.*
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)